python/pyspark/mllib/feature.py

```diff
@@ -120,6 +120,9 @@ def transform(self, vector):
         """
         Applies standardization transformation on a vector.
 
+        Note: In Python, transform cannot currently be used within an RDD
+              transformation or action.  Call transform directly on the RDD instead.
```

Comment:
This line may be too long for Python doc. We use 72 in PySpark, the same as in PEP8.